contracts/src/core/EigenDAServiceManager.sol

```diff
@@ -139,9 +139,9 @@ contract EigenDAServiceManager is EigenDAServiceManagerStorage, ServiceManagerBa
         return batchId;
     }
 
-    /// @notice Returns the block until which operators must serve.
-    function latestServeUntilBlock() external view returns (uint32) {
-        return uint32(block.number) + STORE_DURATION_BLOCKS + BLOCK_STALE_MEASURE;
+    /// @notice Given the current block number, returns the block until which operators must serve.
```

Comment:
"current block" -> "reference block"? An alternative is passing "confirmation block", then the blob will live until `confirmation_block + STORE_DURATION_BLOCKS" which is a simpler definition than using reference block, @gpsanant